topology/src/main/java/io/camunda/zeebe/topology/metrics/TopologyMetrics.java

```diff
@@ -55,6 +61,20 @@ public final class TopologyMetrics {
           .name("cluster_changes_operations_completed")
           .help("Number of completed changes in the current change plan")
           .register();
+  private static final Gauge OPERATION_DURATION =
+      Gauge.build()
```

Comment:
:question: Should this be a histogram? How do you plan to visualize this?

Reply:
I thought a histogram was too much but I'm not sure how to visualize with a gauge. For backups we also used a histogram so I changed it now :+1:

Reply:
We can revisit when we add the visualization.